File: acs/app/src/main/java/com/example/acs/activity/MainActivity.java
package com.example.acs.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.acs.R;
import com.example.acs.activity.Atualizacoes;
import com.example.acs.activity.Infos;
import com.example.acs.activity.Recycle;


public class MainActivity extends AppCompatActivity {

    private Button buttonInfos;

    private Button buttonAtt;

    private Button buttonHist;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        buttonInfos = findViewById(R.id.button1);

        buttonInfos.setOnClickListener( new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent intent = new Intent(getApplicationContext(), Infos.class );
                startActivity(intent);
            }
        });
        buttonAtt = findViewById(R.id.button2);

        buttonAtt.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent intent = new Intent(getApplicationContext(), Atualizacoes.class );
                startActivity(intent);
            }
        });
        buttonHist = findViewById(R.id.buttonHist);

        buttonHist.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent intent = new Intent(getApplicationContext(), Recycle.class );
                startActivity(intent);
            }
        });
    }

    @Override
    protected void onStart() {
        super.onStart();

    }
    @Override
    protected void onResume() {
        super.onResume();

    }
    @Override
    protected void onPause() {
        super.onPause();

    }
        @Override
    protected void onStop() {
        super.onStop();

    }
    @Override
    protected void onRestart() {
        super.onRestart();

    }
    @Override
    protected void onDestroy() {
        super.onDestroy();

    }
}

File: AC02/app/src/main/java/com/example/ac02/MainActivity.kt
package com.example.ac02

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import com.example.ac02.databinding.LoginBinding

class MainActivity : AppCompatActivity() {

    private val binding by lazy {
        LoginBinding.inflate(layoutInflater)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)

        binding.imagem1.setImageResource(R.drawable.mapa_pixel)
        binding.textView.text = "Faça seu Login"
        binding.inputName

        binding.button.setOnClickListener {
            Toast.makeText(this,"Bem Vindo", Toast.LENGTH_SHORT).show()

            if (binding.inputName.inputType == "aluno"){

            }

            val intent = Intent(this, TelaInicialActivity::class.java)
            val params = Bundle()
            params.putString("nome", "aluno")
            params.putString("senha", "impacta")
            intent.putExtras(params)

            val user = binding.inputName.text.toString()
            val password = binding.inputPass.text.toString()
            intent.putExtra("user", user)
            intent.putExtra("senha", password)
            startActivity(intent)

        }
    }
}